stl/inc/ranges

```diff
@@ -33,32 +33,36 @@ namespace ranges {
     template <class _Ty>
     inline constexpr bool _Is_initializer_list = _Is_specialization_v<remove_cvref_t<_Ty>, initializer_list>;
 
-    // clang-format off
+#if _HAS_CXX23
+    _EXPORT_STD template <range _Rng>
+    using const_iterator_t = const_iterator<iterator_t<_Rng>>;
+

```

Comment:
Perhaps it's suitable to implement `const_sentinel_t` right now, since LWG-3770 has been set to Tentatively Ready.